docs/mllib-frequent-pattern-mining.md

```diff
@@ -96,3 +96,76 @@ for (FPGrowth.FreqItemset<String> itemset: model.freqItemsets().toJavaRDD().coll
 
 </div>
 </div>
+
+## Association Rules
+
+Using the `freqItemsets` of an
+[`FPGrowthModel`](api/scala/index.html#org.apache.spark.mllib.fpm.FPGrowthModel)
```

Comment:
This block should go to the `Scala` tab because it contains links to the Scala API doc. We should have a similar one for Java.